From aa7b53cef20298e98208321e9c5120f9bdaa67cc Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Mon, 16 Apr 2012 18:29:49 +0000
Subject: [PATCH 0267/1076] mcorba.lib, mcorbas.lib V 1.0.6.18

Mantis 0018942: mCorba: creation order of servant orb and client orb and synchronization problems

git-svn-id: svn://localhost/SelfServiceCommon/trunk@321 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/cpp/MassaiCorba/src/orbContainer.cpp   | 2 +-
 SelfServiceCommon/Massai/cpp/MassaiCorba/test/src/testcases.txt | 1 -
 2 files changed, 1 insertion(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/src/orbContainer.cpp b/SelfServiceCommon/Massai/cpp/MassaiCorba/src/orbContainer.cpp
index a59ef216..cd84d6bf 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/src/orbContainer.cpp
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/src/orbContainer.cpp
@@ -42,7 +42,7 @@ namespace massaiCorba
     invocationTimeout(getORBInvocationTimeout()),
     numericAddress(useNumericAddress()),
     logging(useLogging()),
-    nestedCalls(allowNestedCallsNestedCalls())
+    nestedCalls(allowNestedCalls())
   {
     if(port == 0)
     {
diff --git a/SelfServiceCommon/Massai/cpp/MassaiCorba/test/src/testcases.txt b/SelfServiceCommon/Massai/cpp/MassaiCorba/test/src/testcases.txt
index d1e04bf7..c72f6c4d 100644
--- a/SelfServiceCommon/Massai/cpp/MassaiCorba/test/src/testcases.txt
+++ b/SelfServiceCommon/Massai/cpp/MassaiCorba/test/src/testcases.txt
@@ -19,7 +19,6 @@ passed
 
 - Test case 2: Nested Upcalls
     start mCorba_testcase2_clnt2
-    set environment CORBA_NO_NESTED_CALLS
     start mCorba_testcase2_clnt1
 
     check thread of client call is not used by servant calls
-- 
2.41.0.windows.1

